Student-Service/src/main/java/tw/waterball/judgegirl/studentservice/domain/repositories/StudentRepository.java

```diff
@@ -34,5 +34,6 @@
 
     void deleteAll();
 
-    List<Student> findAll(int skip, int size);
+    List<Student> findStudents(int skip, int size, boolean isAdmin);
```

Comment:
always put `skip` and `size` as the final two parameters
so the signature should be `findStudents(boolean isAdmin, int skip, int size);`